libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -132,7 +132,7 @@ class DropInNavigationView @JvmOverloads constructor(
         /**
          * Single point of entry for the Mapbox Navigation View.
          */
-        attachCreated(
+        attachStarted(
```

Comment:
I'm not proposing we change this back, but I'm communicating that this was the situation that caused the crash.

The crash is not possible now because of coincidence. The top level Coordinator will only remove the views when the views are destroyed.